packages/api/src/hooks/useCryptoTransactions.ts

```diff
@@ -61,8 +71,49 @@ const useCryptoTransactions = () => {
     const modified_transactions = useMemo(() => {
         if (!transactions || !transactions.length) return undefined;
 
+        const getFormattedConfirmations = (transaction: TModifiedTransaction) => {
```

Comment:
minor/style: as a habit, I tend to leave such functions outside of component - typically its easier to read component itself, and easier to mock/export if needed,

Reply:
Done